mathOperations_hw/index.js

```diff
@@ -0,0 +1,12 @@
+let firstDigit = prompt(`Please enter the first digit.`);
```

Comment:
this is overhead to use template strings in current case
```suggestion
let firstDigit = prompt('Please enter the first digit.');
```